docs/quickstart.md

```diff
@@ -0,0 +1,111 @@
+## Quickstart on a Single Host
```

Comment:
Add a top level Heading that says "# Quickstart"

Then add a table of contents below it that links to the types of deployments:

  - [SingleHost](#single-host)

We can add in the sections for the following as we go:
  - [Docker](#docker)
  - [Kubernetes](#kubernetes)